index.html

```diff
@@ -0,0 +1,69 @@
+               <input type="text" name="search_query" id="search" placeholder="Search">
+        <img id="voice" src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAOEAAADhCAMAAAAJbSJIAAAAeFBMVEX///8AAACGhob5+fk1NTWlpaVHR0eKioq9vb3w8PDn5+egoKCysrIfHx/Dw8Pz8/Pf39+rq6tcXFzo6OiTk5PW1tZPT08YGBg9PT1kZGR1dXVtbW3JyclUVFQmJiY4ODh9fX0uLi4MDAx4eHgbGxuRkZHQ0NBJSUlcX5YGAAAFhUlEQVR4nO2dbXuiOhCGi/iGr2itLepa6e7a//8PTzmtlzUzA0kkJOk+92fE3BAmCSGThwcAAAAAAAAAAAAAAAAAAIB/jsl4ut8Oh8NtMR3nvgvTOulhl9yyH6e+C9Uij6PnhKE38V2wlshOnN7njfwRjivRr2Lqu3h3k21rBZPkJfLbeHhrEPxg7LuQ9zBu9vtg5buY9tQ/glc2vgtqy0xTMEkGvotqx1lbMEkWvgtrQ8q28hKZ7+JasDYRTIa+i2uOXhi9El1ATf8YGkZXT6emgknPd5ENMRZMkrhGU7pt/XfiaveHFoal70KbkFsIJsnZd7ENMI8zFSPfxTagsDL85bvY+iyNOmxX4mkSH+0Ek3jeMS4sDWe+C66NXaCJqUXcWBrGE0xHlobxdE17MIRh8MAQhuEDQxiGDwxhGD4whGH4wBCG4QNDGIYPDGEYPjCEYfjAEIbhA0MYhg8MYRg+MPx3DeP5ngaGMAwfGMIwfGAIw/CBIQzDB4YwDB8YhmyYTgc38CuV2jWc3f7l1O0Kt1Qp05E9ynbNzF/2bK/KUY4T2ij/VrILsG3XPfEJh16Uo5ZOBVXDP2yVGVga8ilOSuUop350nf0jd9DB0vCJO9ly3q2h+nfssk/bNaTs5crUJbduBcljz1/2o5Ug/1CrK9/f3Ao+qKnz+ObCbi33nj2XWuVdr/hWF8DyL+LtQg2fwEUNzHyT0h5q2pmCPWpiZcg+hqTWuE5koz4Vr3xGkqY8dBxC9VNPdXBoV6F2aoQLb5phqILPTJephzlPS1AqfyhUGvOsCs/8icjafndqX6iJLPkAaJCr7YKQbkDtxQv/1yKk6MJxfUPBvnAetYvhPlkm6a8IWWVM8tHVnIZE5Q5yZ6hd/Z1wnJyXleMknEUdiB0djywq1PZJHM2Y1FOpjpKItXZk9R0S3KQm2KTZl0a15M86SX+iXlYxIeCTtiDbf68gqQk7SdJDHjAxg5XuOFHMfEmuEd9JbBsSJd/FQ/XuongH6Rilo4y8JDewnH10Ujb6lXL8pxfIhQ4DSSEkRsIPfjcISo1NBclr19VHU6QzXBvhDnW38Vg3UqDppjrLjE0b89oQJ4+Ha7tgNImt+z7pBdrSNTTEM7UjVPHSMJYlXYuakNQ69M+bWuJ8s74JUL82TcWldbTLnHxM5sfmRyTLZ5tRr9cbbWZ5c8Od0WzunWaOpBFy3nKCVVqvu2ntLyzpTWy3T0yfA+F9iTOYtHptNlbM6TufXCxpGdpLCsi8A3nuYGB4C5f9sS1FLguxh6yR3Bi+HUVOsNsw84X6jqiijWeRnV/1kgqbHRrdH1H33Gldv+gWYNOU9u8L6tk7d1JvH2mwk2hv98SEBVf1PaZQTvnXaXvrh4Z/AznvvKG4QkeKn7fRbsuRM3sDOxwVckgvYrbmr6ZzaeLYU5S5IM7A7MyufCZ+R+V9Vy95kmmvfx8n8hSAd8HaydCtXgU712xJE4Bg/VThvPfUEFifRmXN74MQbJoNLU/ijpWTRa9++iaYna4aX23Pd6vFreZkMfstNA5XPEfR70y0ZtLm/W1RjIpi2290+//woDYSWJrtX6XDe2h7Qdh+NStx8i1EORvvYVVHkFvNpezIzop1aDX0wkxjT04dgmkkKEuzDzB4dqHewE9ydoxuwEuH0y+WjG3267rwGsdWMwtuLk2Hfjwb5x5sHPtBthAi+cjsa/b5KfznTyVd6H/Pvh57fNl0D8uFTutxilXvi3xVyPX1WAyCGkBYk51XdLz0PDiH3bQbQjsCr76L1DK0G9CPa2fVRmAYPzCMHxjGDwzjB4bxA8P4gWH8wDB+YBg/MIwfGMYPDOMHhvHz8w3VLGhJcvxhhoexSlyz9gAAAAAAAAAAAPDNf/LrRrIXyTumAAAAAElFTkSuQmCC" alt="voice-sound">
```

Comment:
Please try to use shorter image addresses, because they make the code easier to read and review.